logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/MessageSender.kt

```diff
@@ -53,20 +52,19 @@ class MessageSenderImpl(
         envelope: MessageEnvelope,
         messageUuid: String,
     ): Either<CoreFailure, Unit> = suspending {
-        val sendResult = messageRepository.sendEnvelope(conversationId, envelope)
-
-        if (sendResult.isRight()) {
-            return@suspending Either.Right(Unit)
-        }
-
-        when (val failure = sendResult.value) {
-            is SendMessageFailure.ClientsHaveChanged -> {
-                messageSendFailureHandler.handleClientsHaveChangedFailure(failure).flatMap {
-                    getRecipientsAndAttemptSend(conversationId, messageUuid)
+        messageRepository.sendEnvelope(conversationId, envelope).coFold(
```

Comment:
Unrelated changes?

Reply:
this is for a different PR i will revert the changes